src/eduid_common/config/parsers/ini.py

```diff
@@ -29,7 +29,8 @@ def __init__(self, config_file_name, config_environment_variable=None):
         :return: IniConfigParser object
         :rtype: IniConfigParser
         """
-
+        if config_environment_variable == None:
```

Comment:
Can't we move this to the get_config_file method?
Let self.config_environment_variable be None if config_environment_variable is omitted.

Reply:
In get_config_file we can do:
```
if self.config_environment_variable:
    env_file = os.environ.get(self.config_environment_variable, None)
    if os.path.exists(env_file):
        return env_file

if os.path.exists(self.config_file_name):
    return self.config_file_name
```

Reply:
Done! :)